tests/suites/test_suite_pk.function

```diff
@@ -1569,3 +1700,220 @@ exit:
     PSA_DONE();
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PSA_CRYPTO_C */
+void pk_get_psa_attributes(int pk_type, int from_pair,
+                           int usage_arg,
+                           int to_pair, int expected_alg)
+{
+    mbedtls_pk_context pk;
+    mbedtls_pk_init(&pk);
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    psa_key_usage_t usage = usage_arg;
+
+    MD_OR_USE_PSA_INIT();
+
+    psa_key_type_t expected_psa_type = 0;
+    if (!pk_setup_for_type(pk_type, from_pair, &pk, &expected_psa_type)) {
```

Comment:
`TEST_EQUAL`? (only if the above comment about the return value of `pk_setup_for_type` is accepted, of course)